docs/manual-EN/server-administration/storage-balance.md

```diff
@@ -0,0 +1,219 @@
+
+```
+(user@127.0.0.1) [(none)]> balance data
+
+==============
+| ID         |
+==============
+| 1570761786 |
+--------------
+Got 1 rows (Time spent: 14378/15410 us)
+```
+
+Check the progress of balance using command `balance data $id`.
+
+```
+(user@127.0.0.1) [(none)]> balance data 1570761786
+
+===============================================================================
+| balanceId, spaceId:partId, src->dst                           | status      |
+===============================================================================
+| [1570761786, 1:1, 192.168.8.210:34600->192.168.8.210:44920]   | succeeded   |
+-------------------------------------------------------------------------------
+| [1570761786, 1:1, 192.168.8.210:34700->192.168.8.210:34920]   | succeeded   |
+-------------------------------------------------------------------------------
+| [1570761786, 1:1, 192.168.8.210:34500->192.168.8.210:34800]   | succeeded   |
+-------------------------------------------------------------------------------
+| [1570761786, 1:2, 192.168.8.210:34600->192.168.8.210:44920]   | in progress |
+-------------------------------------------------------------------------------
+| [1570761786, 1:2, 192.168.8.210:34700->192.168.8.210:34920]   | in progress |
+-------------------------------------------------------------------------------
+| [1570761786, 1:2, 192.168.8.210:34500->192.168.8.210:34800]   | in progress |
+-------------------------------------------------------------------------------
+| [1570761786, 1:3, 192.168.8.210:34600->192.168.8.210:44920]   | succeeded   |
+-------------------------------------------------------------------------------
+...
+| Total:189, Succeeded:170, Failed:0, In Progress:19, Invalid:0 | 89.947090%  |
+-------------------------------------------------------------------------------
+
+Got 190 rows (Time spent: 5454/11095 us)
+```
+
+**_Explanations on the returned results:_**
+
+
+- The first column is a specific balance task. Take 1570761786, 1:88, 192.168.8.210:34700->192.168.8.210:35940 for example
+
+  - **1570761786** is the balance id
+  - **1:88**, 1 is the spaceId, 88 is the moved partId
+  - **192.168.8.210:34700->192.168.8.210:3594**, data migrated from 192.168.8.210:34700 to 192.168.8.210:35940
+-  The second column shows the state of the task, there are four states at present
+  - Succeeded: Operation succeed
+  - Failed: Operation failed
+  - In progress: Operating
+  - Invalid: Invalid task
+
+The last row is the summary of the tasks, some partitions are yet to be migrated.
+### Step 4 Migration confirmation
+In most cases, data migration is a long process. Fortunately, the existing services won't be affected. Once migration is done, the progress will show 100%. Rerun `balance data` to fix failed tasks. If they can't be fixed after several attempts, please contact us at [GitHub](https://github.com/vesoft-inc/nebula/issues).
```

Comment:
In most cases, data migration will take hours or even days. During the migration,  Nebula service are not affected. 

You can retry `balance data` to fix a failed task.